apps/src/componentLibrary/checkbox/checkbox.module.scss

```diff
@@ -20,7 +20,6 @@
 
   span {
     margin-bottom: 0;
-    line-height: initial;
```

Comment:
I see removing this line seems to increase the `line-height`. Just curious why you included this change. Also, where is this`initial` value defined? Thanks!

Reply:
Yep, it increases the line height to be what initially it should've be :D. Reason I added this `initial` value was to fix vertical alignment for previous font we were using.

Styles for this span are set in tsx part, we're using `Typography` `bodyText-${size}` styles here (they have line height)

Reply:
Thanks for the explanation!